### PR TITLE
@joeyAghion => See if this fixes the memcache test failures

### PR DESCRIPTION
**Pull request by mzikherman**
Let's see

### CODE REVIEW

File: test/lib/auth/middleware.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ describe("auth middleware", () => {
 
       expect(middleware.authenticateWithUser({ user: {} })).toBeFalsy()
 
-      process.env.NODE_NEV = "test"
+      process.env.NODE_ENV = "test"
```
Comment (mzikherman) [inline test/lib/auth/middleware.js]: This was the critical fix. Thanks @joeyAghion

Comment (orta) [inline test/lib/auth/middleware.js]: :D

**Comment by mzikherman** [comment]
Alright merging!

**Comment by joeyAghion** [comment]
👍 thanks

**Pull request merged**